content/ja/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -71,74 +142,62 @@ front-proxy-ca          Dec 28, 2029 23:36 UTC   9y              no
 
 また、証明書が外部管理されている場合、kubeadmはユーザーに通知します。この場合、ユーザーは証明書の更新を手動または他のツールを使用して管理する必要があります。
 
-{{< warning >}}
-`kubeadm`は外部CAによって署名された証明書を管理することができません。
-{{< /warning >}}
-
-{{< note >}}
-kubeadmは`/var/lib/kubelet/pki`以下にあるローテート可能な証明書でkubeletの[証明書の自動更新](/docs/task/tls/certificate-rotation/)を構成するので`kubelet.conf`は上記のリストに含まれません。
-
+kubeadmは`/var/lib/kubelet/pki`以下にある更新可能な証明書でkubeletの[証明書の自動更新](/docs/tasks/tls/certificate-rotation/)を構成するので`kubelet.conf`は上記のリストに含まれません。
 期限切れのkubeletクライアント証明書を修復するには、[Kubelet クライアント証明書のローテーションに失敗しました](/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm/#kubelet-client-cert)を参照ください。
-{{< /note >}}
-
-{{< warning >}}
-kubeadm version 1.17より前の`kubeadm init`で作成したノードでは、`kubelet.conf`の内容を手動で変更しなければならないという[bug](https://github.com/kubernetes/kubeadm/issues/1753)が存在します。
 
+{{< note >}}
+kubeadm version 1.17より前の`kubeadm init`で作成したノードでは、`kubelet.conf`の内容を手動で変更しなければならないという[バグ](https://github.com/kubernetes/kubeadm/issues/1753)が存在します。
 `kubeadm init`が終了したら、`client-certificate-data`と`client-key-data`を置き換えて、ローテーションされたkubeletクライアント証明書を指すように`kubelet.conf`を更新してください。
 
 ```yaml
 client-certificate: /var/lib/kubelet/pki/kubelet-client-current.pem
 client-key: /var/lib/kubelet/pki/kubelet-client-current.pem
 ```
-{{< /warning >}}
+{{< /note >}}
 
 ## 証明書の自動更新
 
 kubeadmはコントロールプレーンの[アップグレード](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)時にすべての証明書を更新します。
 
 この機能は、最もシンプルなユースケースに対応するために設計されています。
-証明書の更新に特別な要件がなく、Kubernetesのバージョンアップを定期的に行う場合(各アップグレードの間隔が1年未満)、kubeadmがクラスターを最新かつ適度に安全に保つための処理を行います。
-
-{{< note >}}
-安全性を維持するために、クラスターを頻繁にアップグレードすることがベストプラクティスです。
-{{< /note >}}
+証明書の更新に特別な要件がなく、Kubernetesのバージョンアップを定期的に行う場合(各アップグレードの間隔が1年未満)、kubeadmは、クラスターを最新に保ち、適切なセキュリティを確保します。
 
 証明書の更新に関してより複雑な要求がある場合は、`--certificate-renewal=false`を`kubeadm upgrade apply`や`kubeadm upgrade node`に渡して、デフォルトの動作から外れるようにすることができます。
 
-{{< warning >}}
-kubeadmバージョン1.17より前のバージョンでは、`kubeadm upgrade node`コマンドの`--certificate-renewal`のデフォルト値が`false`になっているという[bug(https://github.com/kubernetes/kubeadm/issues/1818)]問題があります。
-この場合、明示的に`--certificate-renewal=true`を設定する必要があります。
-{{< /warning >}}
-
 ## 手動による証明書更新
 
-`kubeadm certs renew` コマンドを使えば、いつでも証明書を手動で更新することができます。
+適切なコマンドラインオプションを指定して`kubeadm certs renew`コマンドを実行すれば、いつでも証明書を手動で更新できます。
+複数のコントロールプレーンによってクラスターが稼働している場合、全てのコントロールプレーンノード上でこのコマンドを実行する必要があります。
 
-このコマンドは`/etc/kubernetes/pki`に格納されているCA(またはfront-proxy-CA)の証明書と鍵を使って更新を行います。
+このコマンドは`/etc/kubernetes/pki`に格納されているCA(またはfront-proxy-CA)の証明書と鍵を使用して更新を行います。
+
+`kubeadm certs renew`は、属性(Common Name、Organization、SANなど)の信頼できるソースとして、`kubeadm-config` ConfigMapではなく、既存の証明書を使用します。
+それでも、Kubernetesプロジェクトでは、混乱のリスクを避けるために証明書とConfigMap内の関連した値を同期したままにしておくことを推奨しています。
 
 コマンド実行後、コントロールプレーンのPodを再起動する必要があります。
-これは、現在すべてのコンポーネントと証明書について動的な証明書のリロードがサポートされていないため、必要な作業です。
+これは、現在は動的な証明書のリロードがサポートされていないため、すべてのコンポーネントと証明書に対して必要な作業です。
```

Comment:
```suggestion
これは、現在は動的な証明書のリロードが、すべてのコンポーネントと証明書でサポートされているわけではないため、必要な作業です。
```
> This is required since dynamic certificate reload is currently not supported for all components and certificates.

文法的に、「コンポーネントと証明書」は、「サポートされていない」に係っているように思います。
そういう意味では、変更前の翻訳文が近いのですが、部分否定の "not ... all ..." がはっきりとわかるように「～わけではない」の表現にしました。